shared_infra/s3.tf

```diff
@@ -70,6 +70,15 @@ resource "aws_s3_bucket" "mets-ingest" {
 resource "aws_s3_bucket" "wellcomecollection-miro-images-public" {
   bucket = "wellcomecollection-miro-images-public"
   acl    = "public-read"
+
+  lifecycle {
+    prevent_destroy = true
+  }
+}
+
+resource "aws_s3_bucket_policy" "wellcomecollection-miro-images-public" {
+  bucket = "${aws_s3_bucket.wellcomecollection-miro-images-public.id}"
+  policy = "${data.aws_iam_policy_document.wellcomecollection-miro-images-public.json}"
```

Comment:
Mmm. I wonder if this bucket should live in the Loris stack? Because it’s primarily used by Loris?

Not saying it definitely should, just an idea.

Reply:
I'm not sure TBH, I did think about that - my feeling is it probably could be in the Loris stack as we only use it as a container for images that Loris serves (should be another PR though).

Reply:
Actually changed my mind :) as this PR introduces the change it makes sense to address this here.

Reply:
👍